src/components/KStepper/KStep.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+  <li class="k-step">
+    <div
+      :class="{ 'completed': state === 'completed' }"
+      class="k-step-container"
+    >
+      <KStepState :state="state" />
+
+      <div
+        :class="{
+          'bolder': state === 'pending' || state === 'error',
+          'error': state === 'error'
+        }"
+        :style="labelStyle"
+        class="k-step-label px-3"
+      >
+        <KLabel>
+          {{ label }}
+        </KLabel>
+      </div>
+    </div>
+  </li>
+</template>
+
+<script lang="ts">
+import { defineComponent, computed } from 'vue'
+import useUtilities from '@/composables/useUtilities'
+import KLabel from '@/components/KLabel/KLabel.vue'
+import KStepState from '@/components/KStepper/KStepState.vue'
+
+const { getSizeFromString } = useUtilities()
+
+export default defineComponent({
+  name: 'KStep',
+  components: { KLabel, KStepState },
+  props: {
+    label: {
+      type: String,
+      required: true,
+    },
+    state: {
+      type: String,
+      default: 'default',
+      validator: (value: string) => ['default', 'pending', 'completed', 'error'].includes(value),
+    },
+    maxLabelWidth: {
+      type: String,
+      default: '170',
+    },
+  },
+  setup(props) {
+    const labelStyle = computed(() => {
+      return {
+        maxWidth: getSizeFromString(props.maxLabelWidth),
+      }
+    })
+
+    return {
+      labelStyle,
+    }
+  },
+})
+</script>
+
+<style lang="scss" scoped>
+@import '@/styles/variables';
+@import '@/styles/functions';
+
+.k-step {
+  display: list-item;
+  padding: var(--spacing-sm) 0;
+  flex: 1 1 0%;
+
+  &:last-child > .k-step-container::after {
+    display: none;
+  }
+
+  .k-step-container {
+    display: flex;
+    margin: auto;
+    flex-direction: column;
+    align-items: center;
+    padding-bottom: var(--spacing-xxs);
+    position: relative;
+
+    .k-step-label {
+      --KInputLabelWeight: 400;
+      min-width: 100px;
+      text-align: center;
+      padding-top: var(--spacing-sm);
+
+      &.bolder {
+        --KInputLabelWeight: 600;
+      }
+    }
+
+    /**
+     * Divider styles
+     */
+    &::after {
+      content: "";
+      height: 2px;
+      width: 100%;
+      position: absolute;
+      top: calc(#{var(--KStepSize, 24px)} / 2);
+      left: 50%;
+      z-index: 0;
+      background-color: var(--KStepDividerColorDefault, var(--grey-300));
+    }
+
+    &.completed::after {
+      background-color: var(--KStepDividerColorCompleted, var(--teal-300));
```

Comment:
Please remove the KStepper CSS variables; we definitely do not want this level of customization (and they don't exist on the `main` branch)